crates/bevy_reflect/src/path/mod.rs

```diff
@@ -573,7 +571,7 @@ mod tests {
         }
     }
 
-    fn access_field(field: &'static str) -> Access {
+    fn access_field(field: &'static str) -> Access<'static> {
```

Comment:
Why is this lifetime added?

Reply:
This is a new lint added in `rustc`: `named_elided_lifetimes`. Since edition 2018 this should be specified, but it got missed out in the original lints, I think.